### DISCUSSION TITLE
Cannot read properties of undefined (reading 'push') | TypeError: Cannot set properties of undefined (setting '1368')

### DISCUSSION THREAD

**Discussion by utkanpar** [webpack/webpack#15921] [General]
**What is the current behavior?**
We are seeing below error in our react project as well after I modify my code in .tsx file and after HMR reloads the page.

We are not using globalObject: "this" but are defaulting globalObject to "self"

Please check for the error below
![image](https://user-images.githubusercontent.com/69013206/172472286-b97a2840-e2cc-4530-abb8-1d034b084200.png)

The error is coming from jsonp chunk loading:
![image](https://user-images.githubusercontent.com/69013206/172472337-b8ed2b2a-a8bd-437a-b73f-49f4da12bd2b.png)

Looks like variable currentUpdateRuntime, currentUpdate are both undefined.

**What is the expected behavior?**
I would expect these errors to not appear.

**If the current behavior is a bug, please provide the steps to reproduce.**
Here is the reproducible repo with the issue: https://github.com/utkanpar/Webpack-HMR-properties-undefined-error

I have also attached the webpack-config file (https://github.com/utkanpar/Webpack-HMR-properties-undefined-error/blob/main/webpack-config.js) and the yarn.lock file at the root level of the repo

**Other relevant information:**
webpack version: 5.69.1
Node.js version: v12.22.1
Operating System: Windows

**Comment by alexander-akait** [comment]
Sounds like you have wrong order `script` tags in HTML...
